### PR TITLE
Task #209154 Create assessment module in EG

### PR DESCRIPTION
**Pull request by Aar-if**
https://tracker.tekdi.net/issues/209154

### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [ ] Code is formatted as per format decided
* [ ] Updated acceptance criteria in tracker
* [ ] Updated test cases in test-cases-tracker

### CODE REVIEW

File: apps/front-end/src/routes/routes.js [hunk 1]
```diff
@@ -337,5 +339,10 @@ export default [
   { path: "/camps/:id/sessionslist", component: CampSessionsList },
   { path: "/camps/:id/sessionslist/:sessionId", component: CampSession },
 
+  {
+    path: "/assessment/:context/:context_id/:do_id",
+    component: Assessment,
+  },
```
Comment (sagarkoshti1990) [inline apps/front-end/src/routes/routes.js:345]: you add remote but this route not working please check on priority

File: modules/assessment/src/pages/Assessment.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    // }
+    //  else {
+    //   data = {
+        
+    //     status: "completed",
+    //     score: score ? score : 0,
+    //     scoreDetails: JSON.stringify(props),
+        
+    //   };
+    // }
```
Comment (sagarkoshti1990) [inline modules/assessment/src/pages/Assessment.js:83]: Remove

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: apps/front-end/src/routes/routes.js (hunk 1 above)
Comment (Aar-if) [reply apps/front-end/src/routes/routes.js:347]: DONE

File: modules/assessment/src/pages/Assessment.js (hunk 2 above)
Comment (Aar-if) [reply modules/assessment/src/pages/Assessment.js]: DONE

**Pull request merged**